app/lib/file_parse_service.rb

```diff
@@ -110,6 +110,11 @@ def self.run_parse_job(study_file, study, user, reparse: false, persist_on_fail:
               anndata_file: study_file.gs_url, extract: %w[cluster], obsm_keys: [obsm_key],
               file_size: study_file.upload_file_size
             )
+          elsif study_file.needs_raw_counts_extraction?
+            params_object = AnnDataIngestParameters.new(
+              anndata_file: study_file.gs_url, extract: %w[raw_counts], extract_raw_counts: true,
```

Comment:
It seems like it ought to be redundant to specify both `extract: %w[raw_counts]` and `extract_raw_counts: true`.  

Reviewing the code I see why it's currently necessary as a practical matter.  But if add equivalent calls elsewhere or refine AnnDataIngestParameters in the future, then setting `extract_raw_counts: true` if `@extract.include` etc. -- somewhat like [`append_raw_counts_extract`](https://github.com/broadinstitute/single_cell_portal_core/blob/cbefd8b5d56df76839728053fb6ce57e6767725f/app/models/ann_data_ingest_parameters.rb#L82C7-L82C32) -- might be cleaner.

Reply:
Actually on reflection you're correct - it isn't necessary.  As written it only requires `extract: %w[raw_counts]` and the command line will be correctly formatted.